src/metabase/models/recent_views.clj

```diff
@@ -55,8 +55,11 @@
 (defn- duplicate-model-ids
   "Returns a set of IDs of duplicate models in the RecentViews table. Duplicate means that the same model and model_id
    shows up more than once. This returns the ids for the copies that are not the most recent entry."
-  [user-id]
-  (->> (t2/select :model/RecentViews :user_id user-id {:order-by [[:timestamp :desc]]})
+  [user-id context]
+  (->> (t2/select :model/RecentViews
+                  :user_id user-id
+                  :context (name context)
```

Comment:
If you define a transformation on `:model/RecentViews` you can use this as a keyword directly in t2 calls

Reply:
fixed